src/main/java/org/jenkinsci/plugins/saml/OpenSAMLWrapper.java

```diff
@@ -115,6 +116,7 @@ protected SAML2Client createSAML2Client() {
             config.setKeystorePath(KS.getKeystorePath());
             config.setKeystorePassword(KS.getKsPassword());
             config.setPrivateKeyPassword(KS.getKsPkPassword());
+            config.setKeystoreAlias(KS.getKsPkAlias());
```

Comment:
without the key alias the validation of the keystore is a little odd, even do it is tested without the key alias but do not have the key alias forces you to have an only one certificate keystore